flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolInteractionsTest.java

```diff
@@ -80,24 +65,12 @@
 
 	@Test
 	public void testSlotAllocationNoResourceManager() throws Exception {
-		final JobID jid = new JobID();
-
-		try (SlotPool pool = new SlotPoolImpl(
-			jid,
-			SystemClock.getInstance(),
-			TestingUtils.infiniteTime(),
-			TestingUtils.infiniteTime(),
-			TestingUtils.infiniteTime()
-		)) {
 
-			pool.start(JobMasterId.generate(), "foobar", testMainThreadExecutor.getMainThreadExecutor());
-			Scheduler scheduler = new SchedulerImpl(LocationPreferenceSlotSelectionStrategy.createDefault(), pool);
-			scheduler.start(testMainThreadExecutor.getMainThreadExecutor());
+		try (SlotPool pool = createAndSetUpSlotPool(false)) {
```

Comment:
I'd prefer to have a `createAndSetUpSlotPoolWithoutResourceManager()` to replace `createAndSetUpSlotPool(false)`

Reply:
Done.